src/screens/OtherProfile/index.js

```diff
@@ -149,10 +144,9 @@ const OtherProfile = () => {
         setOtherProfileFeed(clonedFeeds, dispatchOtherProfile);
         StorageUtils.otherProfileFeed.setForKey(otherId, JSON.stringify(clonedFeeds));
       }
-      setLoading(false);
       setPostOffset(Number(result.offset));
     } catch (e) {
-      setLoading(false);
+      console.log('error', e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling in this section is insufficient. The catch block only logs the error but does not handle it or provide any feedback to the user. Consider adding user-friendly error handling here.



<!-- This is an auto-generated comment by OSS CodeRabbit -->